### ISSUE TITLE
metrics-server has no chart

### ISSUE THREAD

**Issue by meyskens** [kinvolk/lokomotive#1456]
As per https://github.com/kubernetes-sigs/metrics-server/issues/572 and deprecation of helm/charts there is no official metrics-server chart for the time being, the stable repo of helm is also 403ing now which makes it hard to follow the upgrade steps. We should keep an eye on changes upstream to move to a new chart.

**Comment by invidian** [comment]
There is not much activity, but I'm maintaining the chart for it at https://github.com/flexkube/metrics-server-helm-chart.

**Comment by surajssd** [comment]
There is a PR for metrics server chart: https://github.com/kubernetes-sigs/metrics-server/pull/670.
